app/src/main/java/org/mozilla/tv/firefox/webrender/WebRenderFragment.kt

```diff
@@ -291,12 +292,14 @@ class WebRenderFragment : EngineViewLifecycleFragment(), Session.Observer {
 
         if (event.keyCode == KeyEvent.KEYCODE_DPAD_CENTER && actionIsDown) MenuInteractionMonitor.selectPressed()
 
-        if (event.keyCode == KeyEvent.KEYCODE_MENU && !isUrlEqualToHomepage) {
```

Comment:
This introduces a small bug.

STR
- start app
- press menu button

Expected
- nothing happens

Actual
- blank page is shown

Reply:
Makes sense to me.  `MainActivity#dispatchKeyEvent` seems appropriate.